src/components/Display.js

```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types';
+
+const Display = props => {
+  const { result } = props;
```

Comment:
- [ ] **(OPTIONAL)** You can make this code even shorter by simply destructuring the props directly in the function's argument :smile: